app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -234,6 +223,25 @@ def is_ineligible_or_unsupported_for_property_tax
         StateFile::NjHomeownerEligibilityHelper.determine_eligibility(@intake) != StateFile::NjHomeownerEligibilityHelper::ADVANCE
       end
 
+      def calculate_line_40a
```

Comment:
Moved to be in sequential order by line number, did not change as part of this PR